spec/rubocop/cli_spec.rb

```diff
@@ -1627,6 +1627,23 @@ def abs(path)
       end
     end
 
+    describe '-e/--extra-details' do
+      it 'shows extra details' do
```

Comment:
You should also have an example that adds a `Details` parameter in `.rubocop.yml` and gets it in the output, i.e. has code with the corresponding offense.